chapters/annotations.tex

```diff
@@ -1486,6 +1487,8 @@ \section{Graphical User Interface}\label{annotations-for-the-graphical-user-inte
 The \lstinline!showStartAttribute = true! is primarily intended for non-parameter values and avoids introducing a separate parameter for the \lstinline!start!-attribute of the variable.
 \end{nonnormative}
 
+If the declaration has a modifier for the \lstinline!start!-attribute and does not have \lstinline!showStartAttribute = false! the \lstinline!start!- and \lstinline!fixed!-attributes shall also be shown.
```

Comment:
If we open up for tools to determine what is most appropriate to show by removing default value for showStartAttribute, we should probably not start specifying behavior when showStartAttribute is not set:
```suggestion
If the declaration has a modifier for the \lstinline!start!-attribute and \lstinline!showStartAttribute = true! the \lstinline!fixed!-attribute shall also be shown.
```

Reply:
I will reformulate based on the feedback, but just downgrade from "shall" to "may".

The idea was:
if showStartAttribute=true then recommend showing start and fixed
else if a non-parameter declaration has modifiers for start then
  if showStartAttribute=false -> do not recommend showing start and fixed
  if no showStartAttribute -> recommend showing start and fixed
end if

Reply:
So basically:
```
model M
  Real x(start=1);
equation
  der(x)=1-x;
end M;
```
may generate a dialog for setting the start-value as a convenience, but if you don't want it (perhaps it is a guess-value that hardly matters) then
```
model M2
  Real x(start=1) annotation(Dialog(showStartAttribute=false));
equation
  x^3=1+time;
end M2;
```
will not show a start-attribute for `x`, and since the Dialog-annotation is otherwise empty there is no reason to show  `x` in the parameter-dialog at all.

Note:
- parameters with start-attribute are different (it's complicated), so exclude them from this extra logic
- it the start-value is final it is more complicated, obviously it should not be modified in the parameter dialog, but Dymola can (depending on settings) still show it - that's why I didn't write anything for that

Reply:
The example with `M2` doesn't really make sense the way the annotation is defined:
> If `showStartAttribute = true` the dialog should allow the user to set the `start`- and `fixed`-attributes for the variable instead of the value of the variable.

We have two things that can be modified – the value and the `start`-attribute – and just one annotation.  It generally makes little sense to offer editing of both the value and the `start`-attribute, and hence I find the natural interpretation of `showStartAttribute = false` to be that one should allow editing the value instead of the `start`-attribute.  If we want another interpretation, I think it has to be said explicitly.